lib/grizzly/transports/DTLS.ex

```diff
@@ -52,100 +51,20 @@ defmodule Grizzly.Transports.DTLS do
   end
 
   @impl Grizzly.Transport
-  # Erlang/OTP <= 23.1.x
-  def parse_response(
-        {:ssl, {:sslsocket, {:gen_udp, {_, {{ip, _port}, _}}, :dtls_connection}, _}, bin_list},
```

Comment:
If we actually needed the ip address for something (we don't), the correct way is to use `:ssl.peername/1` with the socket handle (`{:sslsocket, ...}`).